.github/workflows/pss_test.yaml

```diff
@@ -41,35 +54,27 @@ jobs:
         echo "Waiting for pods in auth namespace to become ready..."
         kubectl wait --for=condition=Ready pods --all --timeout=180s -n auth
 
+    # - name: Configure profile-controller's manager configmap with PSS restricted label
+    #   run: |
+    #     CONFIGMAP=$(kubectl get cm -n kubeflow | awk '{print $1}' | grep -e "namespace-labels-data")
+    #     kubectl get cm $CONFIGMAP -n kubeflow -o yaml > temporary.yaml
+    #     sed -i '/app.kubernetes.io\/part-of: "kubeflow-profile"/{s/.*/&\n    pod-security.kubernetes.io\/enforce: "restricted"/}' temporary.yaml
+    #     kubectl apply -f temporary.yaml
+    #     rm temporary.yaml
+
     - name: Install KF Pipelines
       run: ./tests/gh-actions/install_pipelines.sh
 
-    - name: Apply Pod Security Standards baseline levels for static namespaces
-      run: ./tests/gh-actions/enable_baseline_PSS.sh
-
-    - name: Apply Pod Security Standards baseline levels for dynamic namespaces
-      run: |
-        cat << EOF > ./kustomization.yaml
-        apiVersion: kustomize.config.k8s.io/v1beta1
-        kind: Kustomization
-        resources:
-        - apps/profiles/upstream/overlays/kubeflow
-        components:
-        - contrib/security/PSS/dynamic/baseline
-        EOF
-        kubectl apply -k .
-        rm ./kustomization.yaml
-        kubectl -n kubeflow wait --for=condition=Ready pods -l kustomize.component=profiles --timeout 180s
-
-    - name: Unapply applied baseline values
-      run: |
-        NAMESPACES=("istio-system" "auth" "cert-manager" "oauth2-proxy" "kubeflow")
-        for NAMESPACE in "${NAMESPACES[@]}"; do
-          if kubectl get namespace "$NAMESPACE" >/dev/null 2>&1; then
-            kubectl label namespace $NAMESPACE pod-security.kubernetes.io/enforce-
-          fi
-        done
-        sleep 10
+    # - name: Create dynamic user namespace and check for PSS labels present
```

Comment:
Please remove the debugging parts, such that we can merge soon.

Reply:
Debugging parts along with patch for dynamic namespaces have also been removed in the commit https://github.com/kubeflow/manifests/pull/2866/commits/e40311f4032ed586f8144574bb4e8c742687e5ed. I believe that can be done in a new PR to keep things simple.